app/scripts/lib/AbstractPetnamesBridge.ts

```diff
@@ -28,14 +32,17 @@ enum SyncDirection {
 // A list of changes, grouped by type.
 type ChangeList = Record<ChangeType, PetnameEntry[]>;
 
-type AllowedEvents = NameStateChange;
+type PetnamesBridgeAllowedEvents = NameStateChange;
 
-export type PetnamesBridgeMessenger = RestrictedControllerMessenger<
+export type PetnamesBridgeMessenger<
+  Event extends EventConstraint,
```

Comment:
Depending on our TypeScript version, can we use defaults here to assign these both to `never` if not provided, so it's not required on every bridge implementation such as the address book?